<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcRenderDialog</class>
 <widget class="QDialog" name="lcRenderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="topLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="WidthEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="HeightEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="qualityLabel">
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="QualityComboBox">
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="OutputEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="OutputBrowseButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="renderLayout">
       <item>
        <widget class="QProgressBar" name="RenderProgress">
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RenderButton">
         <property name="text">
          <string>Render</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="preview">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>QualityComboBox</tabstop>
  <tabstop>OutputEdit</tabstop>
  <tabstop>OutputBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
